### PR TITLE
Registration page

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
Form without errors
rwanda_register
Form with errors
rwanda_register_errors_1
rwanda_register_errors_2

**Comment by KaitCrawford** [comment]
@Mitso please review

File: gem/styles/gem-rwanda/modules/_m-profiles.scss [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
   &__form {
     padding: 0 $spacer-mid;
+    & input {
+      border-radius: 0px;
+    }
+    & label {
+      text-transform: capitalize;
+      font-weight: bold;
+    }
+    & ul.errorlist, & span.error {
```
Comment (Mitso) [inline gem/styles/gem-rwanda/modules/_m-profiles.scss]: @KaitCrawford Look great! Can you please add the pink or pink dark color for validation messages and remove padding on the left.

**Comment by KaitCrawford** [comment]
New error messages in form
rwanda_register_errors_3

**Pull request merged**